src/main/java/mate/academy/spring/mapper/impl/response/OrderResponseMapper.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderResponseMapper implements DtoResponseMapper<OrderResponseDto, Order> {
+    @Override
+    public OrderResponseDto toDto(Order order) {
+        return new OrderResponseDto(order.getId(),
+                order.getTickets()
+                        .stream()
+                        .map(Ticket::getId)
+                        .collect(Collectors.toList()),
+                order.getOrderDate(),
+                order.getUser().getId());
```

Comment:
![image](https://user-images.githubusercontent.com/117590814/219951905-13e9df1b-86e1-42ba-89f4-dc187d39b547.png)
same in all other mappers. just use setters, they are lot more readable